Moonshot/baddies/Baddie.gd

```diff
@@ -47,7 +47,7 @@ func move(delta) -> void:
 func check_player_colision(should_damage_player = false) -> bool:
 	for i in body.get_slide_count():
 		var collision = body.get_slide_collision(i)
-		if collision.collider.name == "Player":
```

Comment:
Fix that stelios suggested to an earlier bug

Reply:
I'd remove this, it's tracked in a separate PR & issue